src/main/java/bio/terra/aws/resource/discovery/FilesystemEnvironmentDiscovery.java

```diff
@@ -61,7 +60,7 @@ private AvroConfiguration readIntoConfiguration(Path path, ObjectMapper mapper)
   }
 
   private Path getVersionSubdirectoryPath() {
```

Comment:
macOS file system is not case sensitive, so this passed locally (it shouldn't have)